Cogs.Publishers/UmlSchemaPublisher.cs

```diff
@@ -0,0 +1,142 @@
+﻿// Copyright (c) 2017 Colectica. All rights reserved
+// See the LICENSE file in the project root for more information.
+using Cogs.Model;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+using System.Xml;
+using System.Diagnostics;
+
+namespace Cogs.Publishers
+{
+    /// <summary>
+    /// Generate an uml schema using the Garden of Eden approach, all elements and type definitions are declared globally
+    /// </summary>
+    public class UmlSchemaPublisher
+    {
+        // path to outer file to read into 
```

Comment:
These comments are great.
In csharp, you can use a triple /// to have the member descriptions added to the documentation. Visual studio will auto complete these for you after the first ///

```csharp
/// <summary>
/// path to outer file to read into
/// </summary>
public string CogsLocation { get; set; }
```